devices/moes.js

```diff
@@ -151,4 +153,45 @@ module.exports = [
         toZigbee: [],
         exposes: [e.battery(), e.illuminance(), e.illuminance_lux().withUnit('lx'), e.humidity(), e.temperature()],
     },
+    {
+        zigbeeModel: ['b6wax7g0'],
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_b6wax7g0'}],
+        model: 'BRT-100-TRV',
+        vendor: 'Moes',
+        description: 'Thermostatic Radiator Valve',
+        whiteLabel: [{vendor: 'Moes', model: 'BRT-100-TRV'}],
+        ota: ota.zigbeeOTA,
+        onEvent: tuya.onEventSetLocalTime,
+        supports: 'thermostat, temperature',
+        fromZigbee: [fz.ignore_basic_report, fz.ignore_tuya_set_time, fz.moesS_thermostat],
+        toZigbee: [tz.moesS_thermostat_current_heating_setpoint, tz.moesS_thermostat_child_lock,
+            tz.moesS_thermostat_window_detection, tz.moesS_thermostat_temperature_calibration,
+            tz.moesS_thermostat_temperature_calibration, tz.moesS_thermostat_system_mode,
+            tz.moesS_thermostat_boost_heating, tz.moesS_thermostat_boostHeatingCountdownTimeSet,
+            tz.moesS_thermostat_eco_temperature, tz.moesS_thermostat_max_temperature, 
+            tz.moesS_thermostat_min_temperature, tz.moesS_thermostat_schedule, 
+            tz.moesS_thermostat_moesSecoMode, tz.moesS_thermostat_boost_heating_countdown],
+        exposes: [
+            e.battery(), e.child_lock(),
+            exposes.climate()
+                .withLocalTemperature(ea.STATE).withSetpoint('current_heating_setpoint', 5, 35, 0.5, ea.STATE_SET)
+                .withLocalTemperatureCalibration(ea.STATE_SET),
+            exposes.enum('mode_selection', ea.STATE_SET, ['PROGRAMMING', 'MANUAL', 'TEMPORARY_MANUAL', 'HOLIDAY'])
+                .withDescription('MANUAL MODE ☝ - In this mode, the device executes manual temperature setting. '+
+                'When the set temperature is lower than the "minimum temperature", the valve is closed (forced closed). ' +
+                'PROGRAMMING MODE ⏱ -In this mode, the device executes a preset week programming temperature time and temperature. ' +
+                'HOLIDAY MODE ⛱ - In this mode, for example, the vacation mode is set for 10 days and the temperature is set' +
+                'to 15 degrees Celsius. After 10 days, the device will automatically switch to programming mode. ' +
+                'TEMPORARY MANUAL MODE - In this mode, ☝ icon will flash. At this time, the device executes the manually set ' +
+                'temperature and returns to the weekly programming mode in the next time period. '),
+            exposes.enum('programming_mode', ea.STATE).withDescription('PROGRAMMING MODE ⏱ -In this mode, the device executes a ' +
```

Comment:
There is no converter for the key `programming_mode`, so this doesn't work

Reply:
> Hello @Koenkk
> I also have another problem solution, maybe this one is better? Or?
> Create and insert a new preset.
> 
> ```
> ----------------------	
> add node_modules\zigbee-herdsman-converters\lib\exposes.js
> ----------------------
> 
>     mode_selection: () => new Enum('mode_selection', access.STATE_SET, ['PROGRAMMING', 'MANUAL', 'TEMPORARY_MANUAL', 'HOLIDAY']),
> ```

I asked last time, you couldn't answer me. How should I actually do it?
Create and insert a new preset?

add to node_modules\zigbee-herdsman-converters\lib\exposes.js

`mode_selection: () => new Enum('mode_selection', access.STATE_SET, ['programming', 'manual', 'temporary_manual', 'holiday']),`

It's okay like that?

Reply:
> There is no converter for the key `programming_mode`, so this doesn't work

this is only for control.

![Bildschirmfoto vom 2021-06-07 17-47-11](https://user-images.githubusercontent.com/45510877/121050245-b5075a80-c7b8-11eb-816c-8c4986636231.png)

![Bildschirmfoto vom 2021-06-07 17-38-26](https://user-images.githubusercontent.com/45510877/121051211-9190df80-c7b9-11eb-8bc3-5225354709b0.png)

Reply:
Or can you build a converter for the key programming_mode? Do you have any ideas?

Reply:
I think it's better to name the "programming_mode" for example "shedule" like in TS0601 Tuya TRV. I think it says more about operating mode than "programming_mode". And also change "mode_selection" to "preset" - for one standard.

Reply:
> I think it's better to name the "programming_mode" for example "shedule" like in TS0601 Tuya TRV. I think it says more about operating mode than "programming_mode". And also change "mode_selection" to "preset" - for one standard.

I don't think so. With Moes-TVR names are "programming_mode" and "mode_selection" that is not a Tuya, exactly as in the original instructions. See below. Why should you change? It's about "converter", function as such, but not about names.

![Bildschirmfoto vom 2021-06-08 07-24-24](https://user-images.githubusercontent.com/45510877/121128066-110ac700-c82b-11eb-8abd-42bbceabad39.png)

Reply:
Instead of `mode_selection` I would prefer to name it `preset` (like other thermostats), otherwise this cannot be set from Home Assistant.